packages/gatsby-plugin-offline/README.md

```diff
@@ -29,34 +29,42 @@ and AppCache setup by changing these options so tread carefully.
 
 ```javascript
 const options = {
-  staticFileGlobs: [
-    `${rootDir}/**/*.{woff2}`,
-    `${rootDir}/commons-*js`,
-    `${rootDir}/app-*js`,
+  staticFileGlobs: files.concat([
     `${rootDir}/index.html`,
     `${rootDir}/manifest.json`,
     `${rootDir}/manifest.webmanifest`,
     `${rootDir}/offline-plugin-app-shell-fallback/index.html`,
-  ],
+    // Webpack chunks aren't always detected by `getResourcesFromHTML`
+    // since they're sometimes loaded from other chunks.
+    `${rootDir}/+([1234567890])-*.js`,
```

Comment:
We don't want to precache every single webpack chunk — this could lead to precaching megabytes of JS on larger sites.

Reply:
Ah right - I had some offline issues (blank pages) when these files weren't being cached, but if there are any problems after I've removed this line I'll try and find another way around them.

Reply:
Good news, I'm no longer having problems after removing this line! No further changes needed